recipes/ipart/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: ipart
+  version: 2.0.1
+
+source:
+  url: https://github.com/ihesp/IPART/archive/v2.0.1.tar.gz
+  sha256: f07275dd94568cc0da2d9f6b6582068682716124a72410182ca2b37692bc6678
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python 3.7*
```

Comment:
Do we need to limit this to `python 3.7`? If not, please unpin this. If 3.7 is a minimum requirement, change to `python >=3.7`

Reply:
@synapticarbors Thanks SO MUCH! I implemented your corrections and it now passes the checks.